apis/kueue/v1beta1/admissioncheck_types.go

```diff
@@ -86,6 +86,13 @@ type AdmissionCheckStatus struct {
 	// +patchStrategy=merge
 	// +patchMergeKey=type
 	Conditions []metav1.Condition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
+
+	// singleInstannceInClusterQueue indicates if the AdmissionCheck should be the only one managed by
+	// the same Controller in a ClusterQueue.
+	// Having multiple AdmissionChecks managed by the same Controller of which at least one has this
+	// set to true will cause the ClusterQueue to be marked as Inactive.
+	// +optional
+	SingleInstannceInClusterQueue *bool `json:"singleInstannceInClusterQueue,omitempty"`
```

Comment:
Here, what's meant by `instance`? The `instance` means admissionCheck?

Reply:
If admission check `ac1`  and `ac2` are borhn managed by the same controller `c1` and at least one of them (`ac1`  ,`ac2`) hase `Status.SingleInstanceInClusterQueue=true` adding them both to the list of admission checks for a ClusterQueue `cq1` will make `q1` inactive.